lib/utils/url_launcher.dart

```diff
@@ -73,7 +73,7 @@ class UrlLauncher with GoToDraftChatMixin {
           return;
         }
       }
-      launchUrlString(url!);
+      launchUrlString('https://$url');
```

Comment:
If `url` already have a scheme, will this work? E.g. "tel:123456" or "mailto:abc@example.com"

Reply:
Don't worry about this case. In this case I was verified `url` with scheme if it was valid to return `url` if not only display text. You can see `RegExp` to verify `url` in here https://github.com/nqhhdev/linkfy_text/blob/9f4b9debbe4da621897ea040f24e66da3c9ba980/lib/src/utils/contants.dart#L1718

Reply:
still not clear to me @nqhhdev